api/src/migrations/1618521340231-UpdateLocationColumns.ts

```diff
@@ -0,0 +1,50 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+const Table = 'court_location';
+
+export class UpdateLocationColumns1618521340231 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // clean table
+    await queryRunner.query(`DELETE FROM ${Table}`);
+
+    // delete column location_number
+    await queryRunner.query(`ALTER TABLE ${Table} 
+        DROP COLUMN location_number;`);
+
+    // add new column location_code
+    await queryRunner.query(`ALTER TABLE ${Table} ADD COLUMN "location_code" VARCHAR NOT NULL`);
+
+    await queryRunner.query(`ALTER TABLE ${Table} ADD COLUMN "location_short_desc" VARCHAR NOT NULL`);
+
+    // add lat, lng
+    await queryRunner.query(`ALTER TABLE ${Table} ADD COLUMN "lat" NUMERIC`);
+    await queryRunner.query(`ALTER TABLE ${Table} ADD COLUMN "lng" NUMERIC`);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    // clean table
+    await queryRunner.query(`DELETE FROM ${Table}`);
+
+    // add column location_number
+    await queryRunner.query(`ALTER TABLE ${Table} 
+     ADD COLUMN location_number int;`);
+
+    //  delete location_code
+    await queryRunner.query(`ALTER TABLE ${Table} DROP COLUMN "location_code"`);
+
```

Comment:
you may need a revert of `await queryRunner.query(`ALTER TABLE ${Table} ADD COLUMN "location_short_desc" VARCHAR NOT NULL`);`

Reply:
Good I will do that